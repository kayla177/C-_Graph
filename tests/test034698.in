ENTITY M50G8MQP name_M50G paper
ENTITY 5QSR8XVF name_5QSR paper
ENTITY XTD2OEWD name_XTD2 journal
ENTITY 38MWXV7A name_38MW institution
ENTITY 7BBXWDRA name_7BBX institution
ENTITY PKRMU5F1 name_PKRM institution
ENTITY ZE3JWMJS name_ZE3J paper
ENTITY 83CRCRWD name_83CR institution
ENTITY HY81ZSLM name_HY81 institution
ENTITY HPGTHA8S name_HPGT paper
RELATIONSHIP 38MWXV7A published_in PKRMU5F1 2.6
RELATIONSHIP 5QSR8XVF collaborated_with PKRMU5F1 8.5
RELATIONSHIP PKRMU5F1 published_in HY81ZSLM 3.5
RELATIONSHIP 7BBXWDRA presented_at PKRMU5F1 0.2
RELATIONSHIP ZE3JWMJS authored_by M50G8MQP 4.3
RELATIONSHIP 83CRCRWD published_in 7BBXWDRA 7.6
RELATIONSHIP XTD2OEWD published_in 83CRCRWD 1.2
RELATIONSHIP XTD2OEWD collaborated_with ZE3JWMJS 9.9
RELATIONSHIP HPGTHA8S cited_by 5QSR8XVF 8.9
RELATIONSHIP HPGTHA8S cited_by 7BBXWDRA 8.2
RELATIONSHIP 7BBXWDRA authored_by 38MWXV7A 0.6
RELATIONSHIP M50G8MQP authored_by PKRMU5F1 2.4
RELATIONSHIP HPGTHA8S presented_at 83CRCRWD 2.6
RELATIONSHIP HPGTHA8S affiliated_with 38MWXV7A 8.8
RELATIONSHIP XTD2OEWD cited_by PKRMU5F1 0.3
PRINT HY81ZSLM
PATH 83CRCRWD 38MWXV7A
DELETE PKRMU5F1
PRINT M50G8MQP
HIGHEST
PRINT 83CRCRWD
DELETE 38MWXV7A
PRINT 83CRCRWD
HIGHEST
DELETE XTD2OEWD
EXIT
