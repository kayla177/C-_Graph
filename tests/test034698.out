success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
PKRMU5F1
83CRCRWD 7BBXWDRA HPGTHA8S 38MWXV7A 24.6
success
ZE3JWMJS
M50G8MQP 5QSR8XVF 40.1
7BBXWDRA XTD2OEWD HPGTHA8S
success
7BBXWDRA XTD2OEWD HPGTHA8S
M50G8MQP 5QSR8XVF 40.1
success
